extensions/amp-a4a/0.1/amp-a4a.js

```diff
@@ -415,13 +436,15 @@ export class AmpA4A extends AMP.BaseElement {
         // If we got as far as getting a URL, then load the ad, but note the
         // error.
         if (this.adUrl_) {
+          this.lifecycleReporter_.sendPing('renderViaIframe');
           this.renderViaIframe_(true);
         }
         throw rendered;
-      };
+      }
       if (!rendered) {
         // Was not AMP creative so wrap in cross domain iframe.  layoutCallback
         // has already executed so can do so immediately.
+        this.lifecycleReporter_.sendPing('renderViaIframe');
```

Comment:
I guess eventually those pings will scatter around anyway. But I'm not very strong about this either. Was just to save one line.